lib/mappers/__tests__/mapper.figma.test.js

```diff
@@ -0,0 +1,42 @@
+const getFixtures = require('./__fixtures__/mapper.figma');
+const getMapper = require('..');
+
+let FIXTURES = {};
+let mapper;
+
+beforeAll(() => {
+  mapper = getMapper('');
+  FIXTURES = getFixtures();
+})
+
+describe('figma mapper test suite', () => {
+  describe('-> #formatColors:', () => {
+    it('Given figma formatted array of JSON colors, should transform to a nicely parsed array', () => {
+        expect(mapper.mapColors(FIXTURES.COLORS)).toMatchSnapshot();
+    })
+  })
+})
+  
+  describe('-> #formatTextStyles:', () => {
```

Comment:
shouldn't these describe blocks be in the describe('figma mapper test suite' block?
Then you could move the before hook there as well:

```js
describe('figma mapper test suite', () => {
	beforeAll(() => {
	  mapper = getMapper('');
	  FIXTURES = getFixtures();
	});

	describe('-> #formatColors:', () => {...});
	describe('-> #formatTextStyles:', () => {...});
});
```